public/app/features/dashboard-scene/utils/utils.ts

```diff
@@ -124,3 +131,19 @@ export function getDashboardSceneFor(sceneObject: SceneObject): DashboardScene {
 
   throw new Error('SceneObject root is not a DashboardScene');
 }
+
+export function getQueryRunnerFor(sceneObject: SceneObject | undefined): SceneQueryRunner | undefined {
```

Comment:
Isn't this the same function as https://github.com/grafana/grafana/pull/74701/files#diff-12bf1773a2885e1fd761149cd7dd4bc4b5d7079ae14d8ff64c654460a00fcef8R250 ?

Reply:
@dprokop yes, will unify that